src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="big"
+      >
+        <div class="icon">
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+            class="search__input big"
+          />
+        </div>
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="small"
+      >
+        <div class="icon-small">
+          <input
+            type="text"
+            data-qa="hover"
+            placeholder="Los Angeles"
+            class="search__input small"
+          />
+        </div>
+      </form>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to improve readability.